notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>1. Pleae, change the notebook title to something more meaningful like: "SITCOM-798_settling_time_after_slewing.ipynb"</p><p> </p><p>2. Break the following phrase down into simpler and more concise sentences. I found it a litte bit hard to follow.</p><p> </p><p>"This notebook will return a FAIL/PASS after the last cell if the RMS of the displacement values with respect to a long term value of positions and rotations measured by the IMS at M1M3 exceed/not exceed the IMS repeatability error in a certain required time range after the slew stop"</p><p> </p><p>Here is my suggestion:</p><p> </p><p>"This notebook evaluates the RMS of displacement values against long-term position and rotation measurements taken by the IMS at M1M3. It returns a PASS if these values are within the IMS repeatability error and a FAIL if they exceed it. The evaluation occurs within a specific time range after the slew stops."</p><p> </p><p>3. I think that is better if you say explicitly what is the time range required. I believe that all that information will be in the technote, but it is also a good a idea to give a bit more of context here in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed